### DISCUSSION TITLE
Welcome to fledge.polyfill Discussions!

### DISCUSSION THREAD

**Discussion by iamnewton** [MagniteEngineering/fledge.polyfill#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brodrigu** [comment]
Brad Rodriguez here 👋🏼 .

I lead privacy and identity engineering projects at Magnite.

**Comment by iamnewton** [comment]
My name is Newton.  I work for [Magnite](https://www.magnite.com/) ([MGNI](https://www.google.com/finance/quote/MGNI:NASDAQ?sa=X&ved=2ahUKEwiKs7WNksrvAhVaIDQIHSZhBcsQ3ecFMAB6BAgJEBo)) alongside @brodrigu in the Identity and Privacy space as a Lead Engineer.

I'm a new father, a husband, a baseball fanatic, fitness nerd, casual gamer, A11Y enthusiast, passionate CSS developer and many more.

My professional profile is on [LinkedIn](https://www.linkedin.com/in/iamnewton/).
